plugin.audio.kvartal/resources/language/resource.language.en_gb/strings.po

```diff
@@ -0,0 +1,38 @@
+
+# Kodi Media Center language file
+# Addon Name: Kvartal
+# Addon id: plugin.audio.kvartal
+# Addon Provider: Ikosse
+msgid ""
+msgstr ""
+"Project-Id-Version: Kodi Addons\n"
+"Report-Msgid-Bugs-To: alanwww1@kodi.org\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Kodi Translation Team\n"
+"Language-Team: English (http://www.transifex.com/projects/p/xbmc-addons/language/en/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: sv_SE\n"
```

Comment:
Why this file is in `en_gb` directory?

Reply:
The reason for this weirdness on my part is that I wanted the labels to be in Swedish only, since the content is mostly in Swedish. I just wasn't sure how to achieve this using the po-files. If I only had the resource.language.sv_ se directory, the menu items got empty labels. Anyhow, I followed your advice and created a separate directory and hopefully everything is ok now.

Thank you for the review!